app/controllers/v1/sessions_controller.rb

```diff
@@ -306,8 +311,13 @@ def user_logout(saml_response)
       end
     end
 
-    def new_stats(type, client_id)
-      tags = ["type:#{type}", VERSION_TAG, "client_id:#{client_id}"]
+    def new_stats(type, client_id, operation)
+      tags = if params[:operation].present?
+               ["type:#{type}", VERSION_TAG, "client_id:#{client_id}", "operation:#{operation}"]
```

Comment:
I know I said we should only add the `operation` tag if it exists, but I looked through some monitors and I don't think an additive tag will break anything. So let's just add `operation:` to tags and don't worry whether it was in the params or not